quickstarts/202-coe-starter-kit/main.tf

```diff
@@ -0,0 +1,167 @@
+terraform {
+  required_providers {
+    powerplatform = {
+      source  = "microsoft/power-platform"
+      version = "2.7.0-preview"
```

Comment:
A small recommendation and update, the latest version is 3.1.0, you can set the version to use the latest one with this configuration:  version = ">= 3.1.0"